lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.scss

```diff
@@ -1,56 +1,77 @@
+$panel-properties-height: 56px !default;
+
 .adf {
     &-metadata-properties {
-        .mat-expansion-panel-header.mat-expanded:hover,
-        .mat-expansion-panel-header.mat-expanded:focus {
-            background: var(--adf-theme-background-hover-color);
+        .mat-expansion-panel:not([class*='mat-elevation-z']) {
+            box-shadow: none;
+            border: 1px solid var(--adf-metadata-property-panel-border-color);
+            border-radius: 12px;
+            margin: 12px;
         }
 
         mat-expansion-panel-header {
-            height: 64px;
+            height: $panel-properties-height;
+            padding: 0 12px;
+            border-radius: 12px 12px 0 0;
+
+            &.mat-expanded {
+                height: $panel-properties-height;
+                border-bottom: 1px solid var(--adf-metadata-property-panel-border-color);
+            }
+
+            .mat-content {
+                display: contents;
+                margin-right: 0;
+            }
 
             .adf-metadata-properties-title {
-                font-weight: normal;
+                font-weight: 700;
                 font-size: 15px;
+                padding-left: 12px;
             }
         }
 
-        .mat-expansion-panel:not([class*='mat-elevation-z']) {
-            box-shadow: none;
+        .mat-expansion-panel-content {
```

Comment:
Can we avoid styling internal mat classes?

Reply:
closing based on https://github.com/Alfresco/alfresco-ng2-components/pull/8995#discussion_r1379736785  comment